### ISSUE TITLE
"Hanja" key keep show and I didn't press any key

### ISSUE THREAD

**Issue by Cierra-Runis** [mulaRahul/keyviz#56]
**Describe the bug**
Just keep showing, I even don't now when it occur. Untill I restart the software it disappear.

**Version**: v*1.0.6

To Reproduce**
I know it's important to reproduce issue, but it just come out.

**Expected behavior**
just let you know~

**Screenshots**
![QQ截图20220916121642](https://user-images.githubusercontent.com/29329988/190556276-a58352ea-868e-4cc6-941a-1917a4e45cce.png)

**Desktop (please complete the following information):**
 - Windows 10

**Comment by mulaRahul** [comment]
Most probably you've pressed **IME Hanja mode** key and _due to some reason_ keyviz got the key ⬇️ **press event** but **wasn't** followed by a key ⬆️ **release event** causing the visualization to be stuck like that. It seems that it's related to #23, #57.

**Comment by Cierra-Runis** [comment]
Yes, I reproduce it when I switch to japaese input method (alt + `), and then release alt key.

https://user-images.githubusercontent.com/29329988/190842282-0b839ddb-1760-4a3e-b9ff-d97753bad625.mp4

**Comment by mulaRahul** [comment]
Hello @Cierra-Runis, can you reproduce the same with [v2.0.0a](https://github.com/mulaRahul/keyviz/releases/tag/v2.0.0a)?

**Comment by Cierra-Runis** [comment]
Yes... 

https://github.com/mulaRahul/keyviz/assets/29329988/c236b277-52e6-49e3-9842-8056bdd45568

It can be reproduced by press `Alt` then press `~`, then release `Alt` and release `~` at last.
